overlord/servicestate/quota_handlers_test.go

```diff
@@ -923,7 +923,36 @@ func (s *quotaHandlersSuite) TestQuotaSnapModifyExistingMixable(c *C) {
 	c.Assert(err, ErrorMatches, `quota group \"mixed-grp\" has mixed snaps and sub-groups, which is not supported anymore. please remove it and create it again to make any modifications`)
 }
 
-func (s *quotaHandlersSuite) TestQuotaSnapSubgroupUnmixable(c *C) {
+func (s *quotaHandlersSuite) TestQuotaSnapCanRemoveMixed(c *C) {
```

Comment:
This test is failing btw

Reply:
Ah thanks for catching that, it passes locally but now explains the pop-ups for password! Because the system calls weren't being mocked. I think I'm finally starting to understand exactly how that works!